orchagent/pfcactionhandler.cpp

```diff
@@ -385,16 +385,18 @@ PfcWdLossyHandler::PfcWdLossyHandler(sai_object_id_t port, sai_object_id_t queue
 {
     SWSS_LOG_ENTER();
 
-    uint8_t pfcMask = 0;
+    uint8_t pfcTxMask = 0;
+    uint8_t pfcRxMask = 0;
 
-    if (!gPortsOrch->getPortPfc(port, &pfcMask))
+    if (!gPortsOrch->getPortPfc(port, &pfcTxMask, &pfcRxMask))
     {
         SWSS_LOG_ERROR("Failed to get PFC mask on port 0x%" PRIx64, port);
     }
 
-    pfcMask = static_cast<uint8_t>(pfcMask & ~(1 << queueId));
+    pfcTxMask = static_cast<uint8_t>(pfcTxMask & ~(1 << queueId));
```

Comment:
Why do we need to modify Tx Mask?  In any case, its a noop if pfcwd is triggered on a lossy priority because the txMask only contains priorities 3 and 4

Reply:
Agree, we don' need it. Fixed to modify only Rx mask.